src/screens/NationCreateContainer/NationCreateScreen/index.js

```diff
@@ -380,7 +366,7 @@ class CreateNation extends NavigatorComponent {
                 name: i18n.t('enums.nation.services.physical'),
               }]}
               uniqueKey='id'
-              ref={component => this.multiGovernanceService = component}
+              ref={(component) => { this.multiGovernanceService = component; }}
```

Comment:
Just a comment, no need to fix that. Since that syntax may be overwhelming sometimes, we enable suppressing that rule (no-return-assign) by parentheses. Which means you can use 
``ref={component => (this.multiGovernanceService = component)}``
 without error.

Reply:
Ok, thanks